.azure/scripts/setup-kubernetes.sh

```diff
@@ -80,6 +81,17 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
     fi
 
     minikube addons enable default-storageclass
+
+    # Add Docker hub credentials to Minikube
+    if [ "$COPY_DOCKER_LOGIN" = "true" ]
+    then
+      set +ex
+
+      docker exec "minikube" bash -c "echo '$(cat $HOME/.docker/config.json)'| sudo tee -a /var/lib/kubelet/config.json > /dev/null && sudo systemctl restart kubelet"
+
+      set -ex
+    fi
+
 	  minikube addons enable registry
```

Comment:
Looks like something you did not break. But could you fix the alignment here?

Reply:
It seems that this is problem on more lines, let me fix it.

Reply:
Thanks